src/tlo/scenario.py

```diff
@@ -382,20 +393,52 @@ def run_sample_by_number(self, output_directory, draw_number, sample_number):
         sample = self.get_sample(draw, sample_number)
         log_config = self.scenario.get_log_config(output_directory)
 
-        logger.info(key="message", data=f"Running draw {sample['draw_number']}, sample {sample['sample_number']}")
-
-        sim = Simulation(
-            start_date=self.scenario.start_date,
-            seed=sample["simulation_seed"],
-            log_config=log_config
+        logger.info(
+            key="message",
+            data=f"Running draw {sample['draw_number']}, sample {sample['sample_number']}",
         )
-        sim.register(*self.scenario.modules())
 
-        if sample["parameters"] is not None:
-            self.override_parameters(sim, sample["parameters"])
-
-        sim.make_initial_population(n=self.scenario.pop_size)
-        sim.simulate(end_date=self.scenario.end_date)
+        # if user has specified a restore simulation, we load it from a pickle file
+        if (
+            hasattr(self.scenario, "resume_simulation")
+            and self.scenario.resume_simulation is not None
+        ):
+            sim = Simulation.load_from_pickle(
```

Comment:
add logger.info message here

Reply:
Added a message here to state loading pickled suspended simulation and specify pickle file path loading from